interp/genintern.ml

```diff
@@ -27,18 +27,20 @@ type glob_sign = {
   genv : Environ.env;
   extra : Store.t;
   intern_sign : intern_variable_status;
+  strict_check : bool;
 }
 
 let empty_intern_sign = {
   intern_ids = Id.Set.empty;
   notation_variable_status = Id.Map.empty;
 }
 
-let empty_glob_sign env = {
+let empty_glob_sign strict env = {
```

Comment:
```suggestion
let empty_glob_sign ~strict env = {
```

Reply:
and the same for other functions  which appear in a mli

Reply:
Agreed